plugins/woocommerce-admin/client/customer-effort-score-tracks/data/actions.js

```diff
@@ -23,25 +23,31 @@ export function setCesSurveyQueue( queue ) {
 /**
  * Add a new CES track to the state.
  *
- * @param {string} action        action name for the survey
- * @param {string} label         label for the snackback
- * @param {string} pageNow       value of window.pagenow
- * @param {string} adminPage     value of window.adminpage
- * @param {string} onsubmitLabel label for the snackback onsubmit
- * @param {Object} props         object for optional props
+ * @param {string} action         action name for the survey
+ * @param {string} title          title for the snackback
```

Comment:
I think that there is [a lint error](https://github.com/woocommerce/woocommerce/actions/runs/3527945823/jobs/5917539720#step:4:375) here.

Reply:
Looks like it needs to be reformatted for an object. Done in 1b088f35c75c98d2db7fb0f013e39dde8948ca83.